src/main/webapp/script.js

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let map;
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
+
```

Comment:
I left a comment on this on Tomi's pr, There's a problem with javascripts ```let``` but i'm not 100% sure what it is. I just know that whenever you define variables you should use a ```var``` let me know if I'm wrong or if you have another take about this

Reply:
If I'm not mistaken, "let" has some restrictions on how you use the variables. I think it's blocked scope, meaning the variable is restricted to be used to the specific scope that was created.

Reply:
Yeah, I think you're right. Disregard what I said...